MinimedKit/PumpManager/MinimedPumpManager.swift

```diff
@@ -16,11 +16,18 @@ public protocol MinimedPumpManagerStateObserver: class {
 }
 
 public class MinimedPumpManager: RileyLinkPumpManager {
+    
+    public static let managerIdentifier = "Minimed500"
+    
+    public var managerIdentifier: String {
+        return MinimedPumpManager.managerIdentifier
```

Comment:
[nit] Any reason to still keep the static around? Or, can this be collapsed into: 
```
public let managerIdentifier = "Minimed500"
```
and remove the static?

Same question for static `localizedTitle`.

Reply:
As with the MockManagers, the static is kept to satisfy other usages internal to this plugin.